portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesInterests.testcase

```diff
@@ -91,19 +55,21 @@ definition {
 
 			ACCards.viewCards(cardTitle = "Interests");
 
-			ACTimeFilter.assertCurrentTimeFilterRange(timeFilterRange = "Last 30 days");
+			ACTimeFilter.assertCurrentTimeFilterRange(
+				cardName = "Interests",
+				timeFilterRange = "Last 30 days");
 
 			ACInterests.assertNoCurrentInterests();
 		}
 
-		task ("View Interests card with top interests") {
+		task ("Assert able to change Interests card time filter") {
 			ACTimeFilter.clickTimeFilterButton(cardName = "Interests");
 
-			ACTimeFilter.setLast24Hours();
+			ACTimeFilter.setYesterday();
 
-			for (var interests : list "site name,sports,technology") {
-				ACInterests.viewInterests(interestName = ${interests});
-			}
+			ACTimeFilter.assertCurrentTimeFilterRange(
+				cardName = "Interests",
+				timeFilterRange = "Yesterday");
```

Comment:
Hey @joshitagaki, I don't think this validation makes sense for the purpose of the test, which is to check the interests on the Top Interests card

Reply:
This test serves two purposes from what I saw on our test map. It would check the Interests would show up on the card as well as able to change the time filter. I made a note on the test map of the change being made here where it no longer checks Interests would appear on the card.